packages/core/android/build.gradle

```diff
@@ -1,40 +1,63 @@
 buildscript {
   // Buildscript is evaluated before everything else so we can't use getExtOrDefault
-  def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['AnalyticsReactNative_kotlinVersion']
+  def kotlin_version = rootProject.ext.has("kotlinVersion") ? rootProject.ext.get("kotlinVersion") : project.properties["AnalyticsReactNative_kotlinVersion"]
 
   repositories {
     google()
-    jcenter()
+    mavenCentral()
```

Comment:
Was curious about this - found https://developer.android.com/build/jcenter-migration which explains why you needed to do this :smile:
(And I guess it also just matches the new template)

Reply:
I was actually surprised that we hadn't done this before! 
I remember it used to be very noisy in analytics-android/kotlin